js/task-tracker.js

```diff
@@ -5,9 +5,9 @@ const currentTasks_container = document.querySelector("#currentTasks"),
     completedTasks_container = document.querySelector("#completedTasks");
 
 const priorities = {
-    h: "High",
-    m: "Medium",
-    l: "Low"
+    h: "high",
+    m: "medium",
+    l: "low"
 };
```

Comment:
Please, extract it in module and import or require it to use in other places

Reply:
The same situation, now it is a static property of the class `Task`